app/templates/index.js

```diff
@@ -0,0 +1,3 @@
+export default function (input) {
```

Comment:
named functions are better for debug, that’s why I switched to them from anonymous https://github.com/iamstarkov/generator-tiny-es-nm/commit/e7fc5e2a98f7cca9e39122df01b0cf904e16a261